readthedocs/doc_builder/backends/mkdocs.py

```diff
@@ -84,7 +84,18 @@ def load_yaml_config(self):
         :raises: ``MkDocsYAMLParseError`` if failed due to syntax errors.
         """
         try:
-            return yaml.safe_load(open(self.yaml_file, 'r'),)
+            config = yaml.safe_load(open(self.yaml_file, 'r'),)
```

Comment:
We can remove that trailing comma

```suggestion
            config = yaml.safe_load(open(self.yaml_file, 'r'))
```